packages/mobile/src/screens/explore-screen/tabs/ForYouTab/LetThemDJScreen.tsx

```diff
@@ -0,0 +1,35 @@
+import Status from 'audius-client/src/common/models/Status'
+import {
+  getCollections,
+  getStatus
+} from 'audius-client/src/common/store/pages/explore/exploreCollections/selectors'
+import { ExploreCollectionsVariant } from 'audius-client/src/common/store/pages/explore/types'
+import { isEqual } from 'lodash'
+
+import { CollectionList } from 'app/components/collection-list'
+import { Screen } from 'app/components/core'
+import { Header } from 'app/components/header'
+import { WithLoader } from 'app/components/with-loader/WithLoader'
+import { useSelectorWeb } from 'app/hooks/useSelectorWeb'
+
+import { LET_THEM_DJ } from '../../collections'
+
+export const LetThemDJScreen = () => {
+  const status = useSelectorWeb(
+    state =>
+      getStatus(state, { variant: ExploreCollectionsVariant.LET_THEM_DJ }),
+    isEqual
+  )
+  const exploreData = useSelectorWeb(state =>
+    getCollections(state, { variant: ExploreCollectionsVariant.LET_THEM_DJ })
+  )
+
+  return (
+    <Screen>
+      <Header text={LET_THEM_DJ.title} />
+      <WithLoader loading={status === Status.LOADING}>
```

Comment:
this is cool! WithLoader implies this is an hoc, will think about a good name... cant really come up with anything atm! im also down to just have 
```
 loading ? <ScreenSpinner/> : <ScreenComponent />
```
just have the spinner with the special styles ya know? but i like this too, just thinking of all the options.

Reply:
Yea I realized that the `With__` name has meaning to it, but was the first thing that I thought of. I'll change it later if we do come up with a better name 👌